components/camel-huawei/camel-huaweicloud-obs/pom.xml

```diff
@@ -53,6 +53,12 @@
       <version>${gson-version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.camel</groupId>
```

Comment:
I don't think you need to declare camel-base

Reply:
Removed camel-base dependency. There was a method from the IOConversion class in camel-base that was being used, which is why the dependency was included. Since we only used one method, we have moved it to a utility class to avoid importing the entire package.